envs/example/defaults.yml

```diff
@@ -107,8 +107,11 @@ nova:
     debug: True
     verbose: True
 
+ceph:
+  enabled: false
+
 cinder:
-  enabled: True
+  enabled: {{ ceph.enabled }}
   volume_type: file
```

Comment:
If we are intending to back cinder w/ Ceph these three lines: volume_type, volume_file, and volume_file_size need to change or be removed to avoid creating another LVM on loopback file.